src/features/device-logs/lib/backends/redis.ts

```diff
@@ -155,7 +155,7 @@ export class RedisBackend implements DeviceLogsBackend {
 	}
 
 	private getKey(ctx: LogContext, suffix = 'logs') {
-		return `device:${ctx.id}:${suffix}`;
+		return `{device:${ctx.id}}:${suffix}`;
```

Comment:
Just for the reference why this is beneficial: 
https://redis.io/docs/reference/cluster-spec/#hash-tags